numpy/core/src/multiarray/arrayobject.c

```diff
@@ -86,16 +86,6 @@ NPY_NO_EXPORT int
 PyArray_SetUpdateIfCopyBase(PyArrayObject *arr, PyArrayObject *base)
 {
     int ret;
-#ifdef PYPY_VERSION
-  #ifndef DEPRECATE_UPDATEIFCOPY
-    #define DEPRECATE_UPDATEIFCOPY
-  #endif
-#endif
-
-#ifdef DEPRECATE_UPDATEIFCOPY 
-    /* TODO: enable this once a solution for UPDATEIFCOPY
-     *  and nditer are resolved, also pending the fix for GH7054
-     */
     /* 2017-Nov-10 1.14 */
```

Comment:
I think this needs an additional comment for the non-pypy deprecation

Reply:
What would you suggest? Leaving the "GH7054" issue reference as a hint?

Reply:
```
/* 2017-11-10 1.14 (only in pypy) */
/* 2018-04-21 1.15 (all python implementations) */
```

Reply:
fixed